clients/command/client.go

```diff
@@ -38,46 +39,57 @@ type CommandClient interface {
 }
 
 type commandRestClient struct {
-	url      string
-	endpoint interfaces.Endpointer
+	urlClient interfaces.URLClient
 }
 
 // NewCommandClient creates an instance of CommandClient
 func NewCommandClient(params types.EndpointParams, m interfaces.Endpointer) CommandClient {
-	c := commandRestClient{endpoint: m}
-	c.init(params)
-	return &c
+	return &commandRestClient{urlClient: urlclient.New(params, m)}
 }
 
-func (c *commandRestClient) init(params types.EndpointParams) {
-	if params.UseRegistry {
-		go func(ch chan string) {
-			for {
-				select {
-				case url := <-ch:
-					c.url = url
-				}
-			}
-		}(c.endpoint.Monitor(params))
-	} else {
-		c.url = params.Url
+func (cc *commandRestClient) Get(deviceId string, commandId string, ctx context.Context) (string, error) {
+	url, err := cc.urlClient.Prefix()
```

Comment:
Variable naming in prior PRs is `urlPrefix`; rename `url` here for consistency.  Applies to any/all calls to Prefix().